File: EIDSS v6/android.java/workspace/EIDSS/src/com/bv/eidss/web/HumanCaseInfoOut.java
package com.bv.eidss.web;

import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

import org.json.JSONException;
import org.json.JSONObject;

public class HumanCaseInfoOut {

    public Long id;
    public String offlineCaseID;
    public String caseID;
    public Date notificationDate;
    public String notificationSentBy;
    public String notificationSentByPerson;

    public String lastErrorDescription;

    public boolean isCreated;
    public boolean isUpdated;
    public boolean isFailed;
    
    public HumanCaseInfoOut(JSONObject json) throws JSONException, ParseException{
        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.US);
        id = json.getLong("id");
    	offlineCaseID = json.getString("offlineCaseID");
    	caseID = json.getString("caseID");
    	notificationDate = format.parse(json.getString("notificationDate").replace('T', ' '));
    	notificationSentBy = json.getString("notificationSentBy");
    	notificationSentByPerson = json.getString("notificationSentByPerson");
    	lastErrorDescription = json.getString("lastErrorDescription");
    	isCreated = json.getBoolean("isCreated");
    	isUpdated = json.getBoolean("isUpdated");
    	isFailed = json.getBoolean("isFailed");
    }
}

File: EIDSS v6/eidss.smartphone.web/Models/HumanCaseInfoOut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace eidss.smartphone.web.Models
{
    public class HumanCaseInfoOut
    {
        public long id { get; set; }
        public string offlineCaseID { get; set; }
        public string caseID { get; set; }
        public DateTime notificationDate { get; set; }
        public string notificationSentBy { get; set; }
        public string notificationSentByPerson { get; set; }

        public string lastErrorDescription { get; set; }

        public bool isCreated { get; set; }
        public bool isUpdated { get; set; }
        public bool isFailed { get; set; }

        public HumanCaseInfoOut(HumanCaseInfoIn hcin)
        {
            id = hcin.id;
            offlineCaseID = hcin.offlineCaseID;
            caseID = hcin.caseID;
            lastErrorDescription = "";
        }
    }
}